app/Models/User.php

```diff
@@ -284,4 +288,76 @@ public function preferredLocale(): string {
     protected function getDefaultGuardName(): string {
         return 'web';
     }
+
+    protected function oAuthClients(): HasMany {
+        return $this->hasMany(OAuthClient::class, 'user_id', 'id');
+    }
+
+    public function selectPersonalData(PersonalDataSelection $personalDataSelection): void {
```

Comment:
Kann man das logisch irgendwo auslagern - z.B. in einen Controller, der nur GDPR Dinge tut?
Das User Model ist so schon viel zu lang.

Reply:
Jain. Ist halt im GDPR-Package so vorgesehen, dass das im User Model implementiert wird. Sonst müsste man eine einzelne Methode in ne extra-Klasse auslagern, die dann dennoch hier aufgerufen werden muss.

Ich glaube, wir sollten an anderer Stelle Komplexität aus diesem Model nehmen, die wir selbst beeinflussen können.